### PR TITLE
Upgraded the script and meta file issues in the current package.

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Resolved the upgrade issues with the recent Core update, including the HandProfile update for Grip strength.

Also created a Platform Services Profile for automatically installing profiles

### CODE REVIEW

**Review by StephenHodgson — DISMISSED** [review]

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
Compiler errors

**Comment by SimonDarksideJ** [comment]
Ok, looking in to this error
> The type or namespace name 'Editor' does not exist in the namespace 'XRTK.Utilities'

The others are due to the SimpleWebXR dependency, which will only be resolved once its repackaged

**Comment by SimonDarksideJ** [comment]
Ok, the WebXR component is fixed and should be able to force merge.  But until the Submodule updates to a clean UPM package, that part won't build @StephenHodgson @Rufus31415

**Comment by SimonDarksideJ** [comment]
This is frustrating, I built the project and had fixed the things the errors it's reporting.  Ok, double checking

**Comment by SimonDarksideJ** [comment]
Yay, Build passed. The COre dependancy was the last wrinkle.  

Ok to merge now @StephenHodgson  / @Rufus31415

File: XRTK.WebXR/Packages/manifest.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "com.unity.test-framework": "1.1.19",
     "com.unity.textmeshpro": "2.1.3",
     "com.unity.ugui": "1.0.0",
-    "com.xrtk.core": "0.2.0",
+    "com.xrtk.core": "0.2.9",
```
Comment (StephenHodgson) [inline XRTK.WebXR/Packages/manifest.json:19]: I think this is up to 0.2.10

File: XRTK.WebXR/Packages/com.xrtk.webxr/package.json [hunk 2]
```diff
@@ -10,18 +10,17 @@
     "mixed",
     "reality"
   ],
-  "version": "0.2.1",
+  "version": "0.2.2",
```
Comment (StephenHodgson) [inline XRTK.WebXR/Packages/com.xrtk.webxr/package.json:13]: We've never actually made a release for this. It should be 0.2.0

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**